static/sass/_pattern_fluid_video.scss

```diff
@@ -0,0 +1,15 @@
+@mixin ubuntu-p-fluid-video {
+  .p-fluid-video {
+    position: relative;
+    padding-bottom: 56.25%; // 16:9
+    height: 0;
+
+    &__iframe {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+    }
+  }
+}
```

Comment:
We have this in Vanilla although slightly hidden to be fair: https://docs.vanillaframework.io/en/utilities/embedded-media